alicloud/cpi.yml

```diff
@@ -0,0 +1,107 @@
+---
+# CPI
+- type: replace
+  path: /releases/-
+  value:
+    name: bosh-alicloud-cpi
+    version: 16
+    url: http://bosh-((region)).oss-((region)).aliyuncs.com/bosh-alicloud-cpi-release-r16.tgz
```

Comment:
wanted to add to bosh.io releases but found out that https://github.com/cloudfoundry-incubator/bosh-alicloud-cpi-release cannot be built with `bosh create-release releases/bosh-alicloud-cpi/bosh-alicloud-cpi-16.yml`. will send credentials to you for the s3 bucket.

Reply:
Hi @cppforlife Thanks for your feedback and looking forward to your credentials.